pkgs/by-name/tr/tracexec/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tracexec";
+  version = "0.1.0";
```

```suggestion
let
  pname = "tracexec";
  version = "0.1.0";
in
rustPlatform.buildRustPackage {
```

Using `rec` is discouraged, please take a look at: https://nix.dev/guides/best-practices#recursive-attribute-set-rec